src/parse.rs

```diff
@@ -1040,6 +1040,7 @@ where
             name,
             with_spread,
             constructor: (),
+            type_: (),
```

Comment:
Looking at the signature I assumed this is untyped, not sure if putting `()` here is correct